nityam_kumar/PostgresAssignment/src/routes/class.ts

```diff
@@ -0,0 +1,17 @@
+import KoaRouter from "koa-router";
+
+import {
+  getClasses,
+  getSchedule,
+  fetchStudentsWithClass,
+} from "../controller/class";
+
+const router = new KoaRouter({ prefix: "/class" });
+
+router.get("/all", getClasses);
+
+router.get("/schedule", getSchedule);
+
+router.get("/students/:cl_id", fetchStudentsWithClass);
```

Comment:
whats are we fetching here, is it students belonging to certain class, then /class/:classId/student